bin/metrics-mesos.rb

```diff
@@ -33,17 +33,7 @@
 require 'socket'
 require 'json'
 
-# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
-MASTER_DEFAULT_PORT = '5050'.freeze
-SLAVE_DEFAULT_PORT = '5051'.freeze
-
 class MesosMetrics < Sensu::Plugin::Metric::CLI::Graphite
-  option :mode,
```

Comment:
Also this makes this a breaking change which I tend to prefer backwards compatible solutions where possible and I do not see the value of anyways.